src/internal/drawImageSync.js

```diff
@@ -21,7 +21,7 @@ export default function (enabledElement, invalidated) {
   const layers = enabledElement.layers || [];
 
   // Check if enabledElement can be redrawn
-  if (!enabledElement.canvas || !(enabledElement.image || layers.length)) {
+  if (!enabledElement.canvas || !enabledElement.image) {

```

Comment:
Checking the length of the layers array is not actually correct. It is absolutely possible and sometimes desirable to create a layer without an image. In this case we still cannot render.

Also, the first thing which will crash is on line 31 below where we're accessing the `stats` property of `enabledElement.image`.

Layers will set `elementElement.image` anytime the `activeLayer` is set which happens automagically when the first layer is added (assuming it has an image).